setup.py

```diff
@@ -11,10 +11,15 @@
         "flake8==3.4.1",
         "mypy==0.641",
     ],
+    'dev': [
+        "bumpversion>=0.5.3,<1",
+        'twine',
```

Comment:
```suggestion
        "twine",
```

_nitpick_ the single quotes really stand out next to the bumpversion's double quotes.

Reply:
copy/pasta